_data/health.yml

```diff
@@ -87,7 +87,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
       - hardware
```

Comment:
```suggestion
      - u2f
```

Reply:
Someone will have to explain to me which hardware constitutes "u2f" and which does not.